azkaban-common/src/main/java/azkaban/constants/ServerProperties.java

```diff
@@ -47,4 +47,12 @@
 
   public static final String IS_METRICS_ENABLED =
       "azkaban.is.metrics.enabled";
+
+  // Hostname for the host, if not specified, canonical hostname will be used
+  public static final String AZKABAN_SERVER_HOST_NAME = "azkaban.server.hostname";
+
+  // Legacy configs section, new configs should follow the naming convention of azkaban.server. for server configs
+
```

Comment:
i see, i thought `for server configs` was an unfinished sentence. thanks @HappyRay for the explanation :)